packages/wrangler/src/logger.ts

```diff
@@ -0,0 +1,50 @@
+import { format } from "node:util";
```

Comment:
Could this use the ESbuild formatter?

Reply:
This is to account for the fact that `console.log` and friends do fancy formatting based on the first string in the parameter list. E.g. `console.log("pre%spost", "foo")` will output `prefoopost`.

But we could then take the output of `format` and pass it through esbuild's `formatMessagesSync()` possibly.

Reply:
> But we could then take the output of format and pass it through esbuild's formatMessagesSync() possibly.

I like this idea, also just saw this `formatMessagesSync()` in the TS interface today and thought you would like the sync version 😆